src/libs/API/parameters/EnablePolicyExpensifyCardsParams.ts

```diff
@@ -0,0 +1,6 @@
+type EnablePolicyExpensifyCardsParams = {
+    policyID: string;
+    enabled: boolean;
+};
```

Comment:
Yeah good point - will add authToken.
Experiencing the same problem with 404 - and that is a reason with toggling off.
We have this line inside useEffect - which should add/remove left menus
<img width="1055" alt="Screenshot 2024-07-01 at 12 33 34" src="https://github.com/Expensify/App/assets/28352309/f06264f8-efc5-4483-82c2-a0696416f2af">
When we fire api call (which is not exists) - field is in pending state - that's why nothing happened immediately 
If i just use here direct key from onyx `policyFeatureStates` and not state `featureStates` it will work
<img width="713" alt="Screenshot 2024-07-01 at 12 35 33" src="https://github.com/Expensify/App/assets/28352309/779c9831-236a-41b4-9918-ffa26ba2dbc7">
Probably when API will be working - everything will be works as well
@allgandalf

Reply:
@MariaHCD can you weigh in here to keep it moving? Thanks!

Reply:
Ah, sorry, I missed this.

The BE API endpoint is not implemented yet. We can focus on the frontend elements / flows for now and then once the BE endpoints are implemented, we can update the FE here and retest the flow.